tests/consensus_tests/utils.py

```diff
@@ -155,6 +155,8 @@ def wait_for_leader_setup(peer_api_uri: str):
             raise Exception(f"Leader was not established in time ({WAIT_TIME_SEC} sec)")
         else:
             time.sleep(RETRY_INTERVAL_SEC)
+    # It seems that waiting for leader to be recognised is not enough
+    time.sleep(3)
```

Comment:
Should we maybe create an issue for this?

Reply:
I believe it's related to the one that Arnaud is working on currently. If we can add multiple peers concurrently without a problem, then we can remove the wait here.

Reply:
https://github.com/qdrant/qdrant/issues/702 pls add some more context to this issue @e-ivkov